src/Model/NumericalPackage.f90

```diff
@@ -235,6 +235,7 @@ subroutine get_block_data(this, tags, lfound, varinames)
     logical :: endOfBlock
     integer(I4B) :: nsize
     integer(I4B) :: j
+    character(len=24)         :: tmpvar
```

Comment:
Why is this 24? Is it a multiple of another constant? Should we add a constant so this is changed automatically when what it depends on is resized?

Reply:
This is defined locally in `subroutine get_block_data`, where `len=24` is the same for `varinames` and `tags`, shown a few lines up on [src/Model/NumericalPackage.f90#L230L232](https://github.com/MODFLOW-USGS/modflow6/blob/04f32c628bf94d36078281b51dc7afb541197a1e/src/Model/NumericalPackage.f90#L230L232). So far, this subroutine is only used in [src/Model/GroundWaterFlow/gwf3npf8.f90#L1635-L1635](https://github.com/MODFLOW-USGS/modflow6/blob/04f32c628bf94d36078281b51dc7afb541197a1e/src/Model/GroundWaterFlow/gwf3npf8.f90#L1635-L1635), which also has `len=24` for these variables. If you can think up a good name for the ConstantsModule, I can add it there.